script/cadence/transactions/proxy-transfer.cdc

```diff
@@ -1,12 +1,12 @@
 import FlowColdStorageProxy from 0xProxy
 
 transaction(sender: Address, receiver: Address, amount: UFix64, nonce: Int64, sig: String) {
-    prepare(payer: AuthAccount) {
+    prepare(payer: auth(BorrowValue)) {
```

Comment:
Since there isn't anything in this prepare block, can just use an unentitled account reference

```suggestion
    prepare(payer: &Account) {
```